### PR TITLE
[DOCU-1719] Add Community Tab to Contributing

### PR DESCRIPTION
**Pull request by falondarville**
### Review

@Kong/team-docs 

### Summary

In order to prep for our upcoming hackathon, we needed some resources:

* Community: provides contributors with information on our docs-as-code process
* Community Expectations: provides contributors with information on our inclusive language and accessibility efforts. Note that I plan on adding more content to the inclusive language section later on but this is a good starting place. 
* Hackathons: provides contributors with general information about effective hackathon participation

In addition to adding three pages, I hid "Terms" and "Plugin docs" from our sidebar nav as they were incomplete and can be confusing to contributors. 

### Reason

We didn't have in-depth information for contributors to be able to join our community. 

### Testing

Community: https://deploy-preview-3124--kongdocs.netlify.app/contributing/community/
Community Expectations: https://deploy-preview-3124--kongdocs.netlify.app/contributing/community-expectations/
Hackathons: https://deploy-preview-3124--kongdocs.netlify.app/contributing/hackathons/

### CODE REVIEW

File: app/contributing/community-expectations.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+* All non-decorative images have useful and descriptive text alternatives. 
+* Avoid screenshots and use descriptive text instead. 
+* Use clear, concise, and easy-to-understand language. 
+* Organize content intentionally and make things easy to find. 
+* Don't get too experimental - stick to content and organization that's predictable. 
+
+For code-based contributions, check the following:
+
+* All clickable elements can be tabbed through in a predictable sequence.
```
Comment (lena-larionova) [inline app/contributing/community-expectations.md:26]: Maybe add a link to guidance for what this is supposed to look like, or link to our own nav files as an example.

File: app/contributing/community.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+From here, determine which applies to you:
+
+* [I know what I want to contribute](#i-know-what-i-want-to-contribute).
+* [I need ideas on what to contribute](#i-need-ideas-on-what-to-contribute).
+
+### I know what I want to contribute
+
+Great! Let's first check to see if someone had the same idea as you. Go to [Issues](https://github.com/Kong/docs.konghq.com/issues) and read through the titles to see if something sounds similar to what you want to contribute or fix. 
+
+If you find an Issue that aligns with what you want to contribute, go ahead and assign it to yourself in the right-side column and add a comment stating that you're going to be working on this task. Then, head to [Make your contribution](#make-your-contribution).
```
Comment (lena-larionova) [inline app/contributing/community.md:36]: I think this needs a line on what happens if you know what you want to do but there are no open issues for it.

File: app/contributing/community.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    c. `git push origin {CONCISE_AND_RELEVANT_BRANCH_NAME}`
+
+4. Navigate to GitHub and you should see a message that prompts you to open a PR (Pull Request). If you cannot locate this message, read the terminal output after you have pushed your code, and you should see a link in there to open a PR. 
+
+5. Fill out the PR with a title (initially populated by your commit message), and all form details. The more thorough and clear you are, the easier it is for us to understand the change and why it's been made. 
+
+6. Submit your PR!
+
+We encourage you to explore the [README](/Kong/docs.konghq.com) more, read through our [contributing guidelines](/contributing/), and also try out setting up your project locally (via instructions in our README). We appreciate your interest and involvement and are looking forward to seeing your future contributions!
```
Comment (lena-larionova) [inline app/contributing/community.md:69]:
```suggestion
We encourage you to explore the [README](https://github.com/Kong/docs.konghq.com/blob/main/README.md) more, read through our [contributing guidelines](/contributing/), and also try out setting up your project locally (via instructions in our README). We appreciate your interest and involvement and are looking forward to seeing your future contributions!
```

File: app/contributing/community.md (hunk 2 above)
Comment (falondarville) [reply app/contributing/community.md:36]: Sure, I think the contributor will not need to open an Issue, and they can instead just start working locally and push their PR. Is this in alignment with your thoughts?

File: app/contributing/community-expectations.md (hunk 1 above)
Comment (falondarville) [reply app/contributing/community-expectations.md:26]: I can link this, which explains it further: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html

File: app/contributing/community.md (hunk 2 above)
Comment (lena-larionova) [reply app/contributing/community.md:36]: Agree - that makes sense to me. If they have an idea with no related open issue, they can absolutely go ahead and start working on it directly.

**Review by lena-larionova — APPROVED** [review]
LGTM!

**Pull request merged**